Need to export module

we need to export the npm module correctly.  Right now if I import the package in another project, the module is not visible.


```bash
yarn add @openbeta/sandbag'
```
in my typescript code

```javascript
import { getScoreForGrade } from '@openbeta/sandbag'
```

error: Cannot find module '@openbeta/sandbag' or its corresponding type declarations.ts(2307

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@openbeta/sandbag",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Rock climbing grades and conversions",
   "repository": "https://github.com/OpenBeta/sandbag.git",
   "main": "./build/index.js",

--- a/package.json
+++ b/package.json
@@ -3,7 +3,8 @@
   "version": "0.0.1",
   "description": "Rock climbing grades and conversions",
   "repository": "https://github.com/OpenBeta/sandbag.git",
-  "main": "build/index.js",
+  "main": "./build/index.js",
+  "types": "./build/index.d.ts",
   "scripts": {
     "prepare": "husky install",
     "lint": "ts-standard",

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,3 +1,6 @@
+
+import { getScoreForGrade } from './vyds'
+
 // Free Climbing Grades
 // YDS
 // British - Tech
@@ -84,3 +87,5 @@ export const freeClimbing = {
 export const bouldering = {
 
 }
+
+export { getScoreForGrade }

--- a/tsconfig.build.json
+++ b/tsconfig.build.json
@@ -1,5 +1,8 @@
 {
     "extends": "./tsconfig.json",
+    "compilerOptions": {
+      "module": "es6",
+    },
     "exclude": [
         "**/*.test.*",
         "**/__mocks__/*",

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -15,7 +15,8 @@
     "sourceMap": true,
     "strictNullChecks": true,
     "allowSyntheticDefaultImports": true,
-    "forceConsistentCasingInFileNames": true
+    "forceConsistentCasingInFileNames": true,
+    "declaration": true
   },
   "include": [
     ".ts",

